src/models/uiConfig.ts

```diff
@@ -0,0 +1,15 @@
+export interface UiConfig {
+	css?: React.CSSProperties;
+	maxZoom?: number;
+	minZoom?: number;
+	zoom?: number;
+	zoomingEnabled?: boolean;
+}
+
+export const defaultStyling: UiConfig = {
```

Comment:
I think if its model you need remove `defaultStyling` from here to [SparqlGraph.tsx](https://github.com/equinor/sparql-graph/pull/64/files#diff-e9c03e956a8896783efb4d9a0cd21ee3da2bd5ed24c6ac145bc504450c0130eb) and also add [uiConfig.ts](https://github.com/equinor/sparql-graph/pull/64/files#diff-d2503062c0792880e2924eb9d9b67187770cb624da60c2d5097a4bc9d595163b) to `index.ts`

Reply:
Agree. Maybe defaultUiConfig.ts?